CMakeOptions.txt

```diff
@@ -12,7 +12,7 @@ option(ENABLE_EXAMPLES  "Build examples"
 option(ENABLE_FAILMALLOC "Build failmalloc test program" ON)
 option(ENABLE_LIB_ONLY  "Build libnghttp2 only.  This is a short hand for -DENABLE_APP=0 -DENABLE_EXAMPLES=0 -DENABLE_HPACK_TOOLS=0")
 option(BUILD_SHARED_LIBS "Build libnghttp2 as a shared library" ON)
-option(BUILD_STATIC_LIBS "Build libnghttp2 in static mode also" OFF)
+option(BUILD_STATIC_LIBS "Build libnghttp2 in static mode also" ON)
```

Comment:
I'm not following exactly why this needs to be on by default. I'm fine either way, but if the reasoning is that testing needs it, all one needs to do is pass `-DBUILD_STATIC_LIBS=ON`. What's the motivation?